crates/swc/tests/tsc-references/symbolProperty33_es2015.2.minified.js

```diff
@@ -1,9 +0,0 @@
-class C1 extends C2 {
```

Comment:
`C2` is referenced before the declaration.
We have a relevant field in the analysis result

Reply:
Ah, TDZ. No other passes consider it, and terser ignores it too. Maybe we could fix this later